.github/workflows/master.yml

```diff
@@ -12,26 +12,20 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14]
+        node: [14]
 
     steps:
     - uses: actions/checkout@v1
     - name: Install Chromium Library Dependencies
       run: |
         sh ./.github/workflows/chromium-lib-install.sh
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js ${{ matrix.node }}
       uses: actions/setup-node@v2
       with:
-        node-version: ${{ matrix.node-version }}
+        node-version: ${{ matrix.node }}
     - name: Installing project dependencies
       run: |
         yarn install --frozen-lockfile && yarn lerna bootstrap
-    - name: Lint
```

Comment:
i figure since we force all branches to up to date anyway before merging, there seems little value in running all these all over again.  I suppose it's more data points to ensure our tests are passing only due to survivorship bias (only representing CI builds) but we run way more CI builds then master builds, so at most master builds would represent 50% of the data sets (one merge === one build) but realistically it probably averages out to about 1.5 - 2 builds per PR.  So master builds probabaly make up about ~25% of all contributed runs.  

Counter point is that a lot goes into running tests, and because computers and software are built by humans and taking into account the laws of physics, sometimes the build does fail inexplicably.  Just like now host every guarantees 100% uptime.  So what do we really value the most with test and builds
- that our logic is sound
- we get reasonable feedback and assurances
- we don't introduce regressions

If things tweak out every once in a while, I think it's OK right?  Our CI runs enough that I think we should fix _it_ if there's any reason for us not to trust it.

Just thinking out loud.